protos/flow.proto

```diff
@@ -371,6 +372,7 @@ message CDCFlowConfigUpdate {
   repeated TableMapping additional_tables = 1;
   uint32 batch_size = 2;
   uint64 idle_timeout = 3;
+  int32 number_of_syncs = 4;
```

Comment:
any reason this is signed, same below

Reply:
int32 is smallest primitive and number of syncs is <= 32 anyways